AWS Docs (#446)
edgelesssys/constellation: docs/docs/workflows/storage.md

You can use them to bring a volume back to a prior state or provision new volumes.
Follow the instructions on how to [install the Constellation CSI driver](#installation) or check out the [repository](https://github.com/edgelesssys/constellation-gcp-compute-persistent-disk-csi-driver) for information about the configuration.

</tabItem>
<tabItem value="aws" label="AWS">

> TODO: Is this already supported? Test or write note here!

Reviewer: @daniel-weisse should know more about this :)

Author: Added disclaimer.

You can use them to bring a volume back to a prior state or provision new volumes.
Follow the instructions on how to [install the Constellation CSI driver](#installation) or check out the [repository](https://github.com/edgelesssys/constellation-gcp-compute-persistent-disk-csi-driver) for information about the configuration.

</tabItem>
<tabItem value="aws" label="AWS">

:::caution

> Storage isn't yet implemented for AWS. If you require this feature, [let us know](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&template=feature_request.md)!

Reviewer 2: Same here, why do we suggest users to create an issue?

> Storage isn't yet implemented for AWS. If you require this feature, [let us know](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&template=feature_request.md)!
> 
> Other CSI drivers that are compatible with Kubernetes on AWS can be used.

Reviewer 3: ```suggestion
Confidential storage isn't yet implemented for AWS. If you require this feature, [let us know](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&template=feature_request.md)!

You may use other (non-confidential) CSI drivers that are compatible with Kubernetes on AWS.
```

    EOF
    ```

</tabItem>
<tabItem value="aws" label="AWS">

:::caution

> Storage isn't yet implemented for AWS. If you require this feature, [let us know](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&template=feature_request.md)!
> 
> Other CSI drivers that are compatible with Kubernetes on AWS can be used.

Reviewer 3: ```suggestion
Confidential storage isn't yet implemented for AWS. If you require this feature, [let us know](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&template=feature_request.md)!

You may use other (non-confidential) CSI drivers that are compatible with Kubernetes on AWS.
```

      encrypted-storage (default) gcp.csi.confidential.cloud  1d
      ```

</tabItem>
<tabItem value="aws" label="AWS">

:::caution

> Storage isn't yet implemented for AWS. If you require this feature, [let us know](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&template=feature_request.md)!
> 
> Other CSI drivers that are compatible with Kubernetes on AWS can be used.

Reviewer 3: ```suggestion
Confidential storage isn't yet implemented for AWS. If you require this feature, [let us know](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&template=feature_request.md)!

You may use other (non-confidential) CSI drivers that are compatible with Kubernetes on AWS.
```
